api-extractor/src/DebugRun.ts

```diff
@@ -32,12 +32,12 @@ extractor.loadExternalPackages('./testInputs/external-api-json');
 extractor.analyze({entryPointFile: './testInputs/example2/index.ts',
   otherFiles: []});
 
-const externalPackageApiRef: IApiDefinitionReference = {
-  scopeName: '',
-  packageName: 'es6-collections',
-  exportName: '',
-  memberName: ''
-};
+const externalPackageApiRef: ApiDefinitionReference = ApiDefinitionReference.createFromParts(
+  '',
+  'es6-collections',
+  '',
+  ''
+);
```

Comment:
This API looks  a little error prone.  Suggest an interface IApiDefinintionReferenceParts